tests/cases/fourslash/importNameCodeFixNewImportFile6.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="fourslash.ts" />
+
+//// [|import { v2 } from './module2';
+////
+//// f1/*0*/();|]
+
+// @Filename: module.ts
+//// export function f1() {}
+//// export var v1 = 5;
+
+// @Filename: module2.ts
+//// export var v2 = 6;
+
+verify.importFixAtPosition([
+`import { v2 } from './module2';

```

Comment:
Would it be interesting to add a pair of tests where two existing imports have different quote types and, in each case, the first one wins?